agenta-web/.env

```diff
@@ -1,4 +1,4 @@
 NEXT_PUBLIC_AGENTA_API_URL=http://localhost
 NEXT_PUBLIC_FF=oss
-TELEMETRY_TRACKING_ENABLED=true
-POSTHOG_SECRET_KEY=phc_hmVSxIjTW1REBHXgj2aw4HW9X6CXb6FzerBgP9XenC7
\ No newline at end of file
+NEXT_PUBLIC_TELEMETRY_TRACKING_ENABLED=true
+NEXT_PUBLIC_POSTHOG_SECRET_KEY=phc_hmVSxIjTW1REBHXgj2aw4HW9X6CXb6FzerBgP9XenC7
```

Comment:
can you please make sure that we are not overwriting the posthog key for ee?

Reply:
> can you please make sure that we are not overwriting the posthog key for ee?

We are not using `NEXT_PUBLIC_POSTHOG_SECRET_KEY` in the `.env` file for the ee version. It's directly used.